communitymanager/src/main/java/org/pesmypetcare/communitymanager/ChatModel.java

```diff
@@ -0,0 +1,36 @@
+package org.pesmypetcare.communitymanager;
+
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+import androidx.lifecycle.ViewModel;
+
+import org.pesmypetcare.communitymanager.datacontainers.MessageDisplay;
+import org.pesmypetcare.communitymanager.managers.ChatManager;
+
+/**
+ * @author Santiago Del Rey
+ */
+public class ChatModel extends ViewModel {
+    private MutableLiveData<MessageDisplay> messageMutableLiveData = new MutableLiveData<>();
+
+    /**
+     * Default constructor.
+     */
+    public ChatModel() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unnecessary constructors - the compiler will generate these for you](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5538337)